app/helpers/callback_helper.rb

```diff
@@ -1,11 +1,19 @@
 module CallbackHelper
-  def format_booking_quotas(quotas_by_day, time_zone = "GMT")
-    quotas_by_day.transform_values do |quotas|
-      quotas.map do |quota|
+  def format_booking_quotas(quotas, time_zone = "GMT")
+    quotas_excluding_today = quotas.reject do |quota|
```

Comment:
Can these two blocks get moved to separate helper methods? They're potentially useful elsewhere and it allows testing the individual bits of functionality.